cypress-tests/cypress/e2e/PaymentMethodListUtils/Commons.js

```diff
@@ -146,6 +147,24 @@ export const create_payment_body_with_currency_country = (
     },
     email: "example@example.com",
   },
+  shipping: {
+    address: {
+      line1: "130",
+      line2: "Koramangala",
+      line3: "Ramnagar",
+      city: "Bengaluru",
+      state: "Karnataka",
+      zip: "560011",
+      country: shippingCountry, // Shipping country from parameter
+      first_name: "John",
+      last_name: "Joseph",
+    },
+    phone: {
+      number: "9888888888",
```

Comment:
```suggestion
      number: "9123456789",
```

Change it else where as well (L145 above).